cluster/charts/crossplane/README.md

```diff
@@ -75,13 +75,20 @@ and their default values.
 | `resourcesCrossplane.limits.memory` | Memory resource limits for Crossplane | `512Mi` |
 | `resourcesCrossplane.requests.cpu` | CPU resource requests for Crossplane | `100m` |
 | `resourcesCrossplane.requests.memory` | Memory resource requests for Crossplane | `256Mi` |
+| `securityContextCrossplane.runAsUser` | Run as user for Crossplane | `2000` |
+| `securityContextCrossplane.runAsGroup` | Run as group for Crossplane | `2000` |
+| `securityContextCrossplane.allowPrivilegeEscalation` | Allow privilege escalation for Crossplane | `false` |
+| `securityContextCrossplane.readOnlyRootFilesystem` | ReadOnly root filesystem for Crossplane | `true` |
```

Comment:
@khos2ow do you think it would make sense to just allow a user to provide any of the `securityContext` values, rather than scoping it to 4 here and 2 below? I think the change in the helm chart below would actually allow them to set values not defined here, but the documentation doesn't indicate that

Reply:
That's a good point and I think it comes down to provide explicit documentation what user can do. But technically Crossplane depends on these 4 and 2 items. That said I think it makes sense to leave them as is (as default needed values) but user can override them or pass additional items of `securityContext` for helm to use.

Reply:
Ah, that is a good point. This documentation is not only showing what the user can set, it is also showing what we set by default 👍 LGTM, thanks!